Tasks/AzureSpringCloudV0/deploymentProvider/azure-arm-spring-cloud.ts

```diff
@@ -182,12 +278,41 @@ export class AzureSpringCloud {
         return deploymentNames;
     }
 
+    protected async getServiceInfo(): Promise<Object> {
+        tl.debug('Obtaining the current service and its properties');
+        const requestUri = this._client.getRequestUri(`${this._resourceId}`, {}, null, API_VERSION);
+
+        try {
+            const response = await this.sendRequest('GET', requestUri);
+            if (response.statusCode == 200) {
+                tl.debug("Service information obtained");
+                return response.body;
+            } else {
+                tl.error(`${tl.loc('UnableToGetServiceInformation')} ${tl.loc('StatusCode')}: ${response.statusCode}`);
+                throw ToError(response);
+            }
+        } catch (error) {
+            throw(error);
+        }
+    }
+
+    /**
+     * Returns the sku of the service
+     */
+    public async getServiceSku(): Promise<String> {
```

Comment:
I will suggest you to use sku.Tier instead of sku.name. 
And this function can be a private one or protected one(if needed for unit tests), right?